pkg/virt-controller/watch/migration_test.go

```diff
@@ -350,10 +348,11 @@ var _ = Describe("Migration watcher", func() {
 	}
 
 	addMigration := func(migration *virtv1.VirtualMachineInstanceMigration) {
-		mockQueue.ExpectAdds(1)
-		migrationSource.Add(migration)
-		mockQueue.Wait()
-		_, err := virtClientset.KubevirtV1().VirtualMachineInstanceMigrations(migration.Namespace).Create(context.Background(), migration, metav1.CreateOptions{})
+		controller.migrationIndexer.Add(migration)
```

Comment:
Just curious but 1c11197fc71084efc725823734e894b76b6683a1 should improve the runtime already, or not?

Reply:
Yes it does! During removal I saw the time decreasing gradually I was removing FakeControllerSource and another jump removing the sync caches.